ramp-for-gutenberg.php

```diff
@@ -22,7 +26,11 @@
 	remove_action( 'admin_init', 'ramp_for_gutenberg_initialize_admin_ui' );
 } );
 
-/** Loading helper **/
+/**
+ * Load Gutenberg via the Gutenberg Ramp plugin.
+ *
+ * @param array|false $criteria Use `false` or [ 'load' => 1 ] to always load, [ 'load' => 0 ] to never load, [ 'post_ids' => [] ] to load for particular posts, and [ 'post_types' => [] ] to load for particular post types.
```

Comment:
Splitting the documentation into multiple lines would really improve the inline-documentation readability!

Reply:
It might also be ok to not introduce this.  Obviously docs are good, but since this is a wrapper function, it should accept the same params as the wrapped function, and we might just be able to rely on the docs for that function, rather than needing to replicate them here.  (They are about to change/expand anyway.)

In any case, this PR should probably only address what it is meant to address -- ie: loading behavior.

On the other hand, not critical either way :)